### ISSUE TITLE
Log timestamps are always the same 

### ISSUE THREAD

**Issue by bertrand-leroy** [drakkan/sftpgo#719]
Hi

The logfile always prints the same timestamp for each log entry. The time reported matches the timestamp when the process was started.
I run sftpgo as a systemd service

Illustration of the issue (time is the same, but in real it's been minutes between all these logs):
<img width="354" alt="sftpgo_logs" src="https://user-images.githubusercontent.com/54276244/153849896-5d7fd514-af52-49ad-8dca-dc46396b56a1.png">

Any idea what might cause that behavior?

`sftpgo.env` contains the following:
```
SFTPGO_LOG_FILE_PATH=/var/lib/sftpgo/sftpgo.log
SFTPGO_LOG_UTC_TIME=true
```

**Comment by drakkan** [comment]
Hi,

this issue should be fixed now, you can download the latest build artifacts from [here](https://github.com/drakkan/sftpgo/actions/runs/1840760225) once CI completes, or disable UTC logging until the next release, the issue does not happen for local time logging

**Comment by drakkan** [comment]
I also pushed the fix to the 2.2.x branch so the deb/rpm that you can download from [here](https://github.com/drakkan/sftpgo/actions/runs/1840796472), once the CI completes, are the v2.2.2 + this fix

**Comment by bertrand-leroy** [comment]
Cheers!
